### PR TITLE
Allow user to change theme on the ui directly

### PR DESCRIPTION
**Pull request by guillaumevincent**

### CODE REVIEW

File: ara/ui/templates/base.html [hunk 1]
```diff
@@ -9,25 +9,20 @@
...
-  {% with "css/pygments-"|add:UI_THEME_VARIANT|add:".css" as pygments_css %}
-    {% static pygments_css as pygments_css_url %}
-    <link rel="stylesheet" href="{% static_url pygments_css_url %}">
-  {% endwith %}
+  <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}">
+  <link rel="stylesheet" href="{% static 'css/ara.css' %}">
+  <link id="bootstrap-dark-css" rel="stylesheet" href="{% static 'css/bootstrap-darkly.min.css' %}" disabled>
+  <link id="pygments-dark-css" rel="stylesheet" href="{% static 'css/pygments-dark.css' %}" disabled>
+  <link id="bootstrap-light-css" rel="stylesheet" href="{% static 'css/bootstrap-default.min.css' %}">
+  <link id="pygments-light-css" rel="stylesheet" href="{% static 'css/pygments-light.css' %}">
```
Comment (dmsimard) [inline ara/ui/templates/base.html:17]: FYI the ``static_url`` macro is somewhat important so we don't break compatibility with the static version generated by ``ara-manage generate /path/to/report``. I'm open to another approach if there is one but I explained the rationale here:
https://github.com/ansible-community/ara/blob/e213db3299b002e6ea55a3b2a629802694ad8822/ara/ui/templatetags/static_url.py#L11-L16

File: ara/ui/templates/base.html [hunk 2]
```diff
@@ -53,15 +54,30 @@
...
+        document.getElementById("pygments-light-css").disabled = true;
+      } else {
+        document.getElementById("bootstrap-dark-css").disabled = true;
+        document.getElementById("pygments-dark-css").disabled = true;
+        document.getElementById("bootstrap-light-css").disabled = false;
+        document.getElementById("pygments-light-css").disabled = false;
+      }
+    });
+  });
+</script>
```
Comment (dmsimard) [inline ara/ui/templates/base.html:81]: Nice! :+1:

File: ara/ui/templates/base.html [hunk 3]
```diff
@@ -9,25 +9,20 @@
...
-  <link rel="stylesheet" href="{% static_url ui_theme_url %}">
-  {% with "css/pygments-"|add:UI_THEME_VARIANT|add:".css" as pygments_css %}
-    {% static pygments_css as pygments_css_url %}
-    <link rel="stylesheet" href="{% static_url pygments_css_url %}">
-  {% endwith %}
+  <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}">
+  <link rel="stylesheet" href="{% static 'css/ara.css' %}">
+  <link id="bootstrap-dark-css" rel="stylesheet" href="{% static 'css/bootstrap-darkly.min.css' %}" disabled>
+  <link id="pygments-dark-css" rel="stylesheet" href="{% static 'css/pygments-dark.css' %}" disabled>
+  <link id="bootstrap-light-css" rel="stylesheet" href="{% static 'css/bootstrap-default.min.css' %}">
```
Comment (dmsimard) [inline ara/ui/templates/base.html:16]: As discussed we should probably replace the default bootstrap by [flatly](https://bootswatch.com/flatly/) so both themes are based off of bootswatch.

Comment (guillaumevincent) [reply ara/ui/templates/base.html:16]: understood, I will fix this

**Comment by dmsimard** [comment]
It's good enough for now, there is some cleanup to do but I can take care of it in another PR.

**Pull request merged**